pkg/operator/configobservation/cloud/observe_cloud_volume_plugin_test.go

```diff
@@ -26,16 +37,35 @@ func TestObserveCloudVolumePlugin(t *testing.T) {
 	}
 	tests := []Test{
 		{
-			"No FG, on Tech Preview platform (AWS)",
+			"Default FG, on GA platform (AWS) post CSI migration",
 			configv1.AWSPlatformType,
-			nil,
+			defaultFeatureGate,
 			map[string]interface{}{},
 			map[string]interface{}{},
 			false,
 		},
 		{
-			"With FG, on Tech Preview platform (AWS)",
-			configv1.AWSPlatformType,
+			"Default FG, on GA platform (Azure) pre CSI migration",
+			configv1.AzurePlatformType,
+			defaultFeatureGate,
+			map[string]interface{}{},
+			map[string]interface{}{
+				"extendedArguments": map[string]interface{}{
+					"external-cloud-volume-plugin": []interface{}{"azure"},
+				}},
+			false,
+		},
```

Comment:
This isn't correct for 4.13, since Azure is tech preview, needs to look more like the GCP one.

Reply:
thanks Joel, i'll try to push an update here